ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/Generate Default Configuration.md

```diff
@@ -0,0 +1,29 @@
+# Generate Default Configuration
```

Comment:
Let's group the knobs for the role itself into a single file which can either be included in the main readme or have it's own place in the menu. (Outside of "Data models"). We may have to move all the data models into their own subfolder so the glob only catch them.
- generate_default_config
- generate_device_documentation
- custom_templates

Reply:
I think I'm following. Ill show you in the morning meeting. Thank you for the pointers